chromium/rules.js

```diff
@@ -42,7 +42,11 @@ function Exclusion(pattern) {
  * @constructor
  */
 function CookieRule(host, cookiename) {
-  this.host_c = new RegExp(host);
+  if (host === ".*" || host === ".+" || host === ".") {
+    this.host_c = trivial_cookie_name_c;
```

Comment:
Could you please make a new `const` above for host rules, e.g. trivial_cookie_host?